src/components/Theme/Theme.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { getThemeName } from 'selectors/theme';
+import { connect } from 'react-redux';
+import BodyClassName from 'react-body-classname';
+
+export const theme = WrappedComponent =>
+  class extends Component {
+    static displayName = 'Theme';
+    static propTypes = {
+      theme: PropTypes.string
+    };
+    render() {
+      return (
+        <BodyClassName className={'theme--' + this.props.theme}>
```

Comment:
I think now that we are using react-body-classname, you can just inline this functionality in App.js - there's a lot of boilerplate required both here and in this component's tests that could be dispensed with since the functionality is only ever used at the application level